src/features/applicationList/ApplicationListContainer.tsx

```diff
@@ -41,6 +48,31 @@ const ApplicationListContainer = () => {
     });
   };
 
+  const [approveOrders, { loading: isSubmittingApproveOrders }] = useMutation<APPROVE_ORDERS, APPROVE_ORDERS_VARS>(
+    APPROVE_ORDERS_MUTATION,
+    {
+      refetchQueries: [getOperationName(BERTH_APPLICATIONS_QUERY) || 'BERTH_APPLICATIONS_QUERY'],
+    }
+  );
+
+  const handleApproveOrders = async (orders: Array<{ orderId: string; email: string }>) => {
+    try {
+      await approveOrders({
+        variables: {
+          input: {
+            orders,
+          },
+        },
+      });
+      hdsToast({
+        type: 'success',
+        labelText: 'applicationList.notifications.offersSent.label',
+        text: 'applicationList.notifications.offersSent.description',
+        translated: true,
+      });
+    } catch (error) {}
```

Comment:
Should an error toast be shown here?

Reply:
Typically these handlers are synchronous and the success toast is triggered in a `then` block, while the error is handled by the generic error handler.